_posts/2023-10-18-cockpit-303.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Cockpit 303
+author: pitti
+date: '2023-10-18'
+tags: cockpit, machines
+slug: cockpit-303
+category: release
+summary: 'Detect missing AppStream metadata, VM creation with SSH keys'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from Cockpit 303 and cockpit-machines 300:
+
+
+## Apps: Detect missing AppStream metadata
+
+If the [AppStream](https://www.freedesktop.org/software/appstream/docs/index.html) metadata is not installed, the "Apps" page cannot show available Cockpit extensions. The page detects this now, and offers the user to install the metadata.
+
+![screenshot of detect missing appstream metadata]({{ site.baseurl }}/images/303-apps-detect-missing-appstream-metadata.png)
+
+![screenshot of detect missing appstream metadata]({{ site.baseurl }}/images/303-apps-detect-missing-appstream-metadata-2.png)
+
+_Thanks to [leomoty](https://github.com/leomoty/) for designing this improvement!_
+
+## Machines: Add SSH keys to VM creation dialog
+
+The provided keys are stored in the `~/.ssh/authorized_keys` file of the designated non-root user, enabling immediate SSH access to the user account right after creating the VM.
```

Comment:
Could use a bit more introductory verbiage before jumping right in here.  Like:

"When creating a new VM, cockpit-machines now offers the option to provide your SSH public key."

Reply:
Good idea, thanks! Added that and adjusted the next sentence accordingly.